locations/spiders/infrastructure/frankston_city_council_disabled_parking_spaces_au.py

```diff
@@ -4,18 +4,16 @@
 
 from locations.categories import Categories, apply_category
 from locations.items import Feature
-from locations.storefinders.opendatasoft_explore import OpendatasoftExploreSpider
+from locations.flatgeobuf_spider import FlatGeobufSpider
 
 
-class FrankstonCityCouncilDisabledParkingSpacesAUSpider(OpendatasoftExploreSpider):
+class FrankstonCityCouncilDisabledParkingSpacesAUSpider(FlatGeobufSpider):
     name = "frankston_city_council_disabled_parking_spaces_au"
-    item_attributes = {"operator": "Frankston City Council", "operator_wikidata": "Q132472668", "nsi_id": "N/A"}
-    api_endpoint = "https://data.frankston.vic.gov.au/api/explore/v2.1/"
-    dataset_id = "frankston-city-council-accessible-parking"
+    item_attributes = {"operator": "Frankston City Council", "operator_wikidata": "Q132472668", "state": "VIC", "nsi_id": "N/A"}
+    allowed_domains = ["connect.pozi.com"]
+    start_urls = ["https://connect.pozi.com/userdata/frankston-publisher/Community/Accessible_Carpark.fgb"]
 
     def post_process_item(self, item: Feature, response: Response, feature: dict) -> Iterable[Feature]:
-        item["ref"] = str(feature["asset_id"])
-        item["state"] = "VIC"
         apply_category(Categories.PARKING_SPACE, item)
-        item["extras"]["parking_space"] = "disabled"
+        apply_category({"parking_space": "disabled"}, item)
```

Comment:
These aren't categories.

Reply:
> These aren't categories.

I've switched back to `item["extras"]["..."] = ""` syntax instead to avoid overloading use of `apply_category`. There will be a fair few other spiders already overloading use of the `apply_category` function.

Perhaps in the future we should implement an `append_tag` function that `apply_category` uses internally, but can also be used for more obscure tags that don't have widespread use yet. Then the `apply_category` function can have type hints changed to only accept `Categories` Enum members. The new `append_tag` function would handle appending multiple values together into the same tag (semicolon delimited) and deduplicating values, as `apply_category` does today.